src/main/scala/com/databricks/labs/overwatch/pipeline/PipelineTargets.scala

```diff
@@ -31,6 +31,15 @@ abstract class PipelineTargets(config: Config) {
       partitionBy = Seq("organization_id")
     )
 
+    lazy private[overwatch] val databricksSqlTarget: PipelineTable = PipelineTable(
+      name = "databricks_sql_snapshot_bronze",
```

Comment:
As mentioned above, we'll need to have a few different bronze tables, so be diligent on the names as they are hard to change later. Once you become more familiar with the APIs and the data structures, you'll be able to nail this. You'll create one `PipelineTable` per bronze table.